test/format/backup.c

```diff
@@ -106,7 +350,11 @@ backup(void *arg)
      * Perform a full backup at somewhere under 10 seconds (that way there's at least one), then at
      * larger intervals, optionally do incremental backups between full backups.
      */
+    block_full = full = true;
     incremental = 0;
+    active_files_init(&active[0]);
+    active_files_init(&active[1]);
```

Comment:
I think we want to call `active_files_free` at the end of this function on both lists.

Reply:
Fixed.